### DISCUSSION TITLE
Invitation link leads to localhost:3000

### DISCUSSION THREAD

**Discussion by manururu** [sinamics/ztnet#219] [Q&A] [answered]
Hello!

⚠️ Please don't be way too mad at me, I'm very new to nginx, hosting and all such web stuff! Neither do I know a thing about node.js😭. I've already tried to formulate my problem both in my mother tongue and in English but here I am, having googled nothing.

Here's the thing: the invitation link to my network is generated with `localhost:3000`:

![image](https://github.com/sinamics/ztnet/assets/107123837/6158692b-2498-49c6-8c11-a6cb2e232967)

I have a VM with public IP running ztnet **standalone** on Debian 11. Since I wanted to reach the web interface via HTTPS using subzone of the domain I already have, I've installed nginx and certbot and created a config file at `/etc/nginx/sites-available`:

```
  server {
    listen 443 ssl; # managed by Certbot
    server_name foo.example.com;

    ssl_certificate /etc/letsencrypt/live/foo.example.com/fullchain.pem; # managed by Certbot
    ssl_certificate_key /etc/letsencrypt/live/foo.example.com/privkey.pem; # managed by Certbot
    include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
    ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot

    location / {
        proxy_pass http://localhost:3000;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection 'upgrade';
        proxy_set_header Host $host;
        proxy_cache_bypass $http_upgrade;
    }


}
server {
    if ($host =  foo.example.com) {
        return 301 https://$host$request_uri;
    } # managed by Certbot


    listen 80;
    server_name foo.example.com;
    return 404; # managed by Certbot


}
```

What am I missing here?

**Comment by sinamics** [comment, marked as answer]
Hi
Change the `NEXTAUTH_URL` in .env file to match your domain name.

`/opt/ztnet/.env`
```yml
NEXTAUTH_URL: "example.com"
```

**Reply by manururu** [reply]
Jeez. Didn't notice just a single line there.

Thank you very much!
